src/TheThingsNetwork.h

```diff
@@ -23,6 +23,19 @@
 #define HEX_CHAR_TO_NIBBLE(c) ((c >= 'A') ? (c - 'A' + 0x0A) : (c - '0'))
 #define HEX_PAIR_TO_BYTE(h, l) ((HEX_CHAR_TO_NIBBLE(h) << 4) + HEX_CHAR_TO_NIBBLE(l))
 
+/* BEGIN TEST */
+
+#ifdef DEBUG
```

Comment:
OK, that works and improves the examples a lot not having to repeat this all the time. Two Q's

- Should we expose this via the library (`ttn.debugPrint`) instead of global scope to prevent conflicts?
- Should we use `debugPrintln` (no upper-case `L` since it's `Serial.println` as well)?